day34/validateIP.py

```diff
@@ -0,0 +1,74 @@
+def validateIP(ip):
+    ipv4 = ip.split(".")
+    ipv6 = ip.split(":")
+    if len(ipv4) == 4 and validateIPv4(ipv4):
+        return "IPv4"
+    elif len(ipv6) == 8 and validateIPv6(ipv6):
+        return "IPv6"
+    else:
+        return "Neither"
+
+def validateIPv4(segments):
```

Comment:
convention is to use `isValidIPv4` if we're returning a boolean, `validateIPv4` seems to imply it'll throw an exception if it's not valid